src/Interpreters/DDLWorker.cpp

```diff
@@ -1388,10 +1388,23 @@ void DDLWorker::markReplicasActive(bool /*reinitialized*/)
             zookeeper->deleteEphemeralNodeIfContentMatches(active_path, active_id);
         }
         Coordination::Requests ops;
+        Coordination::Responses res;
         ops.emplace_back(zkutil::makeCreateRequest(active_path, active_id, zkutil::CreateMode::Ephemeral));
         /// To bump node mtime
         ops.emplace_back(zkutil::makeSetRequest(fs::path(replicas_dir) / host_id, "", -1));
-        zookeeper->multi(ops);
+        auto code = zookeeper->tryMulti(ops, res);
+
+        /// We have this tryMulti for a very weird edge case when it's related to localhost.
+        /// Each replica may have a localhost as hostid and if we configured multiple replicas to add their
+        /// localhosts to some clusters multiple of them may think that they must mark it as active.
+        if (code != Coordination::Error::ZOK)
```

Comment:
Should we allow only `Coordination::Error::ZNODEEXISTS` error here?

Reply:
I'm afraid to break old behavior where it was just `try/catch`. So let's leave it as is.